lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -63,6 +63,12 @@ module.exports = {
             if (event.alb.multiValueHeaders) {
               albObj.multiValueHeaders = event.alb.multiValueHeaders;
             }
+            if (event.alb.targetGroupName) {
+              albObj.targetGroupName = this.validateTargetGroupName(
```

Comment:
Let's not create `validateTargetGroupName` method, but simply validate it inline. If I see correctly there's no re-use use case here.